internal/groupsuffix/groupsuffix.go

```diff
@@ -101,6 +106,50 @@ func mutateOwnerRefs(replaceFunc func(baseAPIGroup, apiGroupSuffix string) (stri
 	}
 }
 
+func mutateTokenCredentialRequestAuthenticatorAPIGroup(apiGroupSuffix string) func(context.Context, kubeclient.RoundTrip) {
```

Comment:
I thought having a helper function read better because now that we have accumulated 4 `MiddlewareFunc`'s, it is helpful to see what each of the sibling `Middleware`'s are doing with just a couple lines of code. But I can change it back to inline.

Reply:
Either way is fine - it mostly a style choice.  When I saw the function was not inlined my intuition was that "it must be used in more than one place" so I went looking for the other caller.

Reply:
Took a pass at this in 0e1336f253f05cace80cc3439540d7aea7782f54.